CONTRIBUTING.md

```diff
@@ -0,0 +1,248 @@
+# Contributing to Lux
+
+Thank you for your interest in contributing to Lux! This document will help
+answer any outstanding questions you may have about the contribution process.
+
+*Please read the [Code of Conduct](./CODE_OF_CONDUCT.md) before you participate
+in community.*
+
+## Contents
+
```

Comment:
The new section looks great. Related, this is pretty interesting: https://github.com/kentcdodds/all-contributors